hackathon_site/registration/admin.py

```diff
@@ -1,6 +1,32 @@
 from django.contrib import admin
-from .models import Application, Team as TeamApplied
+from registration.models import Application, Team as TeamApplied
 
-# Register your models here.
-admin.site.register(Application)
-admin.site.register(TeamApplied)
+
+class ApplicationInline(admin.TabularInline):
+    model = Application
+    autocomplete_fields = ("user",)
+    extra = 0
+
+
+@admin.register(TeamApplied)
+class TeamAppliedAdmin(admin.ModelAdmin):
+    search_fields = ("id", "team_code")
+    inlines = (ApplicationInline,)
+
+
+@admin.register(Application)
+class ApplicationAdmin(admin.ModelAdmin):
```

Comment:
What's the purpose of these classes as opposed to just doing it with `admin.site.register`

Reply:
They're used to customize how the model appears in the admin site: https://docs.djangoproject.com/en/3.1/ref/contrib/admin/#modeladmin-objects.

Doing `admin.site.register(Application, ApplicationAdmin)` does the same thing as the `@admin.register` decorator.